application.rb

```diff
@@ -0,0 +1,193 @@
+require_relative 'services/login_service.rb'
+require_relative 'services/sign_up_service.rb'
+require_relative 'services/task_manager.rb'
+
+class Application
+  def self.boot
+    @@task_manager = TaskManager.new
```

Comment:
no need to create a new object everytimg
```suggestion
    @@task_manager ||= TaskManager.new
```